packages/ember-htmlbars/lib/node-managers/component-node-manager.js

```diff
@@ -26,6 +27,7 @@ function ComponentNodeManager(component, isAngleBracket, scope, renderNode, attr
   this.attrs = attrs;
   this.block = block;
   this.expectElement = expectElement;
+  this[HAS_BLOCK] = undefined;
```

Comment:
This doesn't seem to be used (on the root of the ComponentNodeManager instance).